quadriga/Docker/tomcat/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM tomcat:9.0.95-jdk17
+
+
+ENV JAVA_OPTS="-DappConfigFile=file:/jdk17-path/bin"
+ENV TOMCAT_VERSION 9.0.95
+ENV CATALINA_HOME /usr/local/tomcat
+
+RUN mkdir app
+RUN mkdir data
+RUN mkdir data/config
+RUN mkdir data/files
+
+WORKDIR app
+
+RUN apt clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update && apt-get install git -y && apt install maven -y
+
+RUN git clone -b story/Q20-25 https://github.com/diging/quadriga-2.0.git
+
+RUN cd quadriga-2.0/quadriga && mvn clean package -Dlog.level=info
+RUN cp quadriga-2.0/quadriga/target/vspace.war ../webapps
+
+# download tomcat manager app
+# RUN wget https://mirrors.ocf.berkeley.edu/apache/tomcat/tomcat-$TOMCAT_VERSION/v$TOMCAT_VERSION/bin/apache-tomcat-$TOMCAT_VERSION.zip
+# RUN unzip apache-tomcat-$TOMCAT_VERSION.zip
+RUN wget https://downloads.apache.org/tomcat/tomcat-11/v$TOMCAT_VERSION/bin/apache-tomcat-$TOMCAT_VERSION.tar.gz \
+    && tar -xvzf apache-tomcat-$TOMCAT_VERSION.tar.gz \
+    && mv apache-tomcat-$TOMCAT_VERSION $CATALINA_HOME \
+    && rm apache-tomcat-$TOMCAT_VERSION.tar.gz
+RUN cp -r apache-tomcat-$TOMCAT_VERSION/webapps/manager ../webapps
```

Comment:
if we already use a tomcat base image, we shouldn't have to download tomcat again.